### DISCUSSION TITLE
Okular Dark Theme?

### DISCUSSION THREAD

**Discussion by MillingtonAllen** [i3/i3#4572] [Q&A]
Kubuntu 20.10, i3 WM

I used to use the method described here in KDE:

https://www.reddit.com/r/kde/comments/7ww4ja/changing_okular_colour_scheme/

I've seen the method of changing dark and light colors through accessibility options, but it's not what I'm looking for. While I can change page background via the accessibility settings, it doesn't change the toolbar colors.

I want the pages white, text dark. I want the theme color to change (toolbars, page background) to dark.

It works as expected in a standard Kubuntu install using KDE.  Not sure why it's not working in i3.

**Comment by Airblader** [comment]
The reason it works in KDE is probably because it's a complete desktop environment and as such ships with a lot more tooling not related to window management. You could try running i3 in KDE, or you'll need to figure out which part of KDE is relevant here (I don't know, sorry, but maybe someone does).

i3 is a window manager only and not involved in this at all.

**Reply by MillingtonAllen** [reply]
My guess was that it has something to do with QT theming, but I wasn't  sure.  I've seen suggesting changing the environmental variable to KDE, but that didn't seem to work.

What do you mean when you say "running i3 in KDE"?  I'm assuming this means replacing kwin within KDE somehow, but I'm not sure how to do that.

**Reply by Airblader** [reply]
Yes, that is what I mean. I don't know how well that works since I never did it myself, but see https://github.com/i3/i3/discussions/4537.

**Reply by roke-julian-lockhart** [reply]
> My guess was that it has something to do with QT theming, but I wasn't sure.

@MillingtonAllen, KDE applications adhere to the `QPalette` superset exposed by `KColourScheme`.
